packages/authorization-process/src/model/domain/errors.ts

```diff
@@ -369,3 +370,11 @@ export function eserviceNotDelegableForClientAccess(
     title: "EService not delegable for client access",
   });
 }
+
+export function securityUserNotMember(userId: UserId): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `User ${userId} with SECURITY_ROLE is not a member of the client`,
```

Comment:
Sorry, even better, let's use the actual value used in the token for the role, SECURITY_ROLE is how we call it in the code and only an internal reference (see [here](https://vscode.dev/github/pagopa/interop-be-monorepo/blob/develop/packages/commons/src/auth/authData.ts#L12)):

```suggestion
    detail: `User ${userId} with user role "security" is not a member of the client`,
```

Reply:
Make sense